<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalDataViewModel</class>
 <widget class="QWidget" name="PersonalDataViewModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,7,7">
     <item>
      <widget class="QPushButton" name="saveChangesButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: #4CAF50;
	color: white;
	border: none;
	text-align: center;
	text-decoration: none;
	display: inline-block;
	font-size: 18px;
	cursor: pointer;
	border-radius: 12px;
}

QPushButton:hover{
	background-color: #45a049;
}

QPushButton:pressed{
	background-color: #3e8e41;
}</string>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: gray;
	color: white;
	border: none;
	text-align: center;
	text-decoration: none;
	display: inline-block;
	font-size: 18px;
	cursor: pointer;
	border-radius: 12px;
}

QPushButton:hover{
	background-color: red;
}

QPushButton:pressed{
	background-color: #3e8e41;
}</string>
       </property>
       <property name="text">
        <string>Выйти</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="loadingLayout">
     <item>
      <widget class="QLabel" name="loadingMessage">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font-size: 12pt;
	font-weight: 600;
}

QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}</string>
       </property>
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	font-size: 20pt;
	font-weight: 600;
}

QLineEdit {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font-size: 12pt;
	font-weight: 600;
}

QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}</string>
       </property>
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="surnameEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	font-size: 14pt;
	font-weight: 600;
}

QLineEdit {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font-size: 12pt;
	font-weight: 600;
}

QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}</string>
       </property>
       <property name="text">
        <string>Отчество</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="patronymicEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	font-size: 14pt;
	font-weight: 600;
}

QLineEdit {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font-size: 12pt;
	font-weight: 600;
}

QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}</string>
       </property>
       <property name="text">
        <string>Возраст</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ageEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	font-size: 14pt;
	font-weight: 600;
}

QLineEdit {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font-size: 12pt;
	font-weight: 600;
}

QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}</string>
       </property>
       <property name="text">
        <string>Почта</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="emailEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	font-size: 14pt;
	font-weight: 600;
}

QLineEdit {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font-size: 12pt;
	font-weight: 600;
}

QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}</string>
       </property>
       <property name="text">
        <string>Телефон</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="phoneEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	font-size: 14pt;
	font-weight: 600;
}

QLineEdit {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
